2020-10-28 11:38:04 - Using a Hub instance not created by the HubPipeline is unsupported.
2020-10-28 11:51:05 - Using a Hub instance not created by the HubPipeline is unsupported.
30/11/2020 12:48:33 PM - Two Hubs must not share the same name. 'TNY.NotificationService.SignalR.Hubs.NotificationHub, TNY.NotificationService.SignalR, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' and 'TNY.NotificationService.WebAPI.signalr.hubs.NotificationHub, TNY.NotificationService.WebAPI, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' both share the name 'NotificationHub'.
01/12/2020 3:38:07 PM - An error occurred while deserializing the AppID property of class TNY.NotificationService.BUS.Models.NotificationActivity: Cannot deserialize a 'List<String>' from BsonType 'ObjectId'.
03/12/2020 1:35:14 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T06:34:56.5169891Z", LastUpdateTimestamp: "2020-12-03T06:34:56.5169891Z" }] }.
03/12/2020 1:50:16 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T06:50:09.7481868Z", LastUpdateTimestamp: "2020-12-03T06:50:09.7481868Z" }] }.
03/12/2020 1:52:38 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T06:51:39.4039197Z", LastUpdateTimestamp: "2020-12-03T06:51:39.4039197Z" }] }.
03/12/2020 1:59:28 PM - Connected
03/12/2020 2:00:01 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T07:00:01.1102172Z", LastUpdateTimestamp: "2020-12-03T07:00:01.1102172Z" }] }.
03/12/2020 2:01:56 PM - Disconnected
03/12/2020 2:02:06 PM - Connected
03/12/2020 2:02:36 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T07:02:36.2795339Z", LastUpdateTimestamp: "2020-12-03T07:02:36.2795339Z" }] }.
03/12/2020 2:12:01 PM - Disconnected
03/12/2020 2:12:05 PM - Connected
03/12/2020 2:12:35 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T07:12:35.2217823Z", LastUpdateTimestamp: "2020-12-03T07:12:35.2217823Z" }] }.
03/12/2020 2:14:50 PM - Disconnected
03/12/2020 2:15:02 PM - Connected
03/12/2020 2:15:32 PM - A timeout occured after 30000ms selecting a server using CompositeServerSelector{ Selectors = MongoDB.Driver.MongoClient+AreSessionsSupportedServerSelector, LatencyLimitingServerSelector{ AllowedLatencyRange = 00:00:00.0150000 } }. Client view of cluster state is { ClusterId : "1", ConnectionMode : "Automatic", Type : "Unknown", State : "Disconnected", Servers : [{ ServerId: "{ ClusterId : 1, EndPoint : "Unspecified/localhost:27017" }", EndPoint: "Unspecified/localhost:27017", ReasonChanged: "Heartbeat", State: "Disconnected", ServerVersion: , TopologyVersion: , Type: "Unknown", HeartbeatException: "MongoDB.Driver.MongoConnectionException: An exception occurred while opening a connection to the server. ---> System.Net.Sockets.SocketException: No connection could be made because the target machine actively refused it 127.0.0.1:27017
   at System.Net.Sockets.Socket.InternalEndConnect(IAsyncResult asyncResult)
   at System.Net.Sockets.Socket.EndConnect(IAsyncResult asyncResult)
   at System.Threading.Tasks.TaskFactory`1.FromAsyncCoreLogic(IAsyncResult iar, Func`2 endFunction, Action`1 endAction, Task`1 promise, Boolean requiresSynchronization)
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<ConnectAsync>d__7.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.TcpStreamFactory.<CreateStreamAsync>d__4.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
   --- End of inner exception stack trace ---
   at MongoDB.Driver.Core.Connections.BinaryConnection.<OpenHelperAsync>d__51.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<InitializeConnectionAsync>d__32.MoveNext()
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.TaskAwaiter.ValidateEnd(Task task)
   at MongoDB.Driver.Core.Servers.ServerMonitor.<HeartbeatAsync>d__34.MoveNext()", LastHeartbeatTimestamp: "2020-12-03T07:15:31.4547344Z", LastUpdateTimestamp: "2020-12-03T07:15:31.4547344Z" }] }.
03/12/2020 2:17:38 PM - Disconnected
03/12/2020 2:17:40 PM - Connected
03/12/2020 2:17:40 PM - No leftover notif
03/12/2020 2:26:48 PM - Scheduled Sender running
03/12/2020 2:27:48 PM - Scheduled Sender running
03/12/2020 2:28:11 PM - Object reference not set to an instance of an object.
03/12/2020 2:28:48 PM - Scheduled Sender running
03/12/2020 2:35:45 PM - Scheduled Sender running
03/12/2020 2:36:45 PM - Scheduled Sender running
03/12/2020 2:37:19 PM - Connected
03/12/2020 2:37:19 PM - No leftover notif
03/12/2020 2:37:45 PM - Scheduled Sender running
03/12/2020 2:38:51 PM - Scheduled Sender running
03/12/2020 2:40:31 PM - Scheduled Sender running
03/12/2020 2:41:47 PM - Scheduled Sender running
03/12/2020 2:52:51 PM - Not found: cat - Updating to Recipients
03/12/2020 2:52:51 PM - Not found: dog - Updating to Recipients
03/12/2020 2:52:54 PM - Not found: cat - Updating to Recipients
03/12/2020 2:52:54 PM - Not found: dog - Updating to Recipients
03/12/2020 2:52:54 PM - Scheduled Sender running
03/12/2020 2:54:31 PM - Scheduled Sender running
03/12/2020 3:38:47 PM - Not found: cat - Updating to Recipients
03/12/2020 3:39:06 PM - Not found: dog - Updating to Recipients
03/12/2020 3:39:19 PM - Not found: cat - Updating to Recipients
03/12/2020 3:39:22 PM - Not found: dog - Updating to Recipients
03/12/2020 3:39:31 PM - Scheduled Sender running
03/12/2020 3:54:37 PM - Not found: cat - Updating to Recipients
03/12/2020 3:54:37 PM - Not found: dog - Updating to Recipients
03/12/2020 3:54:39 PM - Not found: cat - Updating to Recipients
03/12/2020 3:54:39 PM - Not found: dog - Updating to Recipients
03/12/2020 3:54:39 PM - Scheduled Sender running
03/12/2020 3:55:47 PM - Not found: cat - Updating to Recipients
03/12/2020 3:55:47 PM - Not found: dog - Updating to Recipients
03/12/2020 3:55:49 PM - Not found: cat - Updating to Recipients
03/12/2020 3:55:49 PM - Not found: dog - Updating to Recipients
03/12/2020 3:55:49 PM - Scheduled Sender running
03/12/2020 3:58:35 PM - Not found: cat - Updating to Recipients
03/12/2020 3:58:35 PM - Not found: dog - Updating to Recipients
03/12/2020 3:59:19 PM - Scheduled Sender running
03/12/2020 4:00:19 PM - Scheduled Sender running
03/12/2020 4:01:19 PM - Scheduled Sender running
03/12/2020 4:01:31 PM - Connected
03/12/2020 4:01:31 PM - Pushing leftover notif
03/12/2020 4:02:19 PM - Scheduled Sender running
05/12/2020 1:17:11 PM - Not found: cat - Updating to Recipients
05/12/2020 1:17:11 PM - A write operation resulted in an error.
  E11000 duplicate key error collection: NotificationDB.Recipients index: _id_ dup key: { _id: "cat" }
05/12/2020 1:17:17 PM - Connected
05/12/2020 1:17:17 PM - No leftover notif
05/12/2020 1:17:20 PM - Sending notif to: cat - cid: 36682562-8021-42da-bbb8-548d36b8de16
05/12/2020 1:17:20 PM - Not found: dog - Updating to Recipients
05/12/2020 1:24:01 PM - Disconnected
29/01/2021 5:51:11 PM - Connected
29/01/2021 5:51:11 PM - No leftover notif
29/01/2021 8:27:10 PM - Connected
29/01/2021 8:27:10 PM - No leftover notif
30/01/2021 1:23:48 AM - Not found: 123 - Updating to Recipients
30/01/2021 1:28:50 AM - Not found: 123 - Updating to Recipients
30/01/2021 1:29:03 AM - Connected
30/01/2021 1:29:03 AM - Pushing leftover notif
30/01/2021 1:34:41 AM - Disconnected
30/01/2021 1:36:20 AM - Not found: 123 - Updating to Recipients
30/01/2021 1:36:20 AM - A write operation resulted in an error.
  E11000 duplicate key error collection: NotificationDB.Recipients index: _id_ dup key: { _id: "123" }
30/01/2021 2:00:36 AM - Connected
30/01/2021 2:00:36 AM - No leftover notif
30/01/2021 2:04:30 AM - Disconnected
30/01/2021 2:04:45 AM - Not found: 123 - Updating to Recipients
30/01/2021 2:04:45 AM - A write operation resulted in an error.
  E11000 duplicate key error collection: NotificationDB.Recipients index: _id_ dup key: { _id: "123" }
30/01/2021 2:05:36 AM - Not found: 123 - Updating to Recipients
30/01/2021 2:06:00 AM - A write operation resulted in an error.
  E11000 duplicate key error collection: NotificationDB.Recipients index: _id_ dup key: { _id: "123" }
30/01/2021 2:06:27 AM - Disconnected
30/01/2021 2:06:36 AM - Connected
30/01/2021 2:06:36 AM - No leftover notif
30/01/2021 2:06:39 AM - Disconnected
30/01/2021 2:06:39 AM - Connected
30/01/2021 2:06:39 AM - No leftover notif
30/01/2021 2:06:41 AM - Disconnected
30/01/2021 2:06:42 AM - Connected
30/01/2021 2:06:42 AM - No leftover notif
01/02/2021 12:31:05 AM - Unable to determine the serialization information for x => x.
01/02/2021 1:51:13 AM - Connected
01/02/2021 1:51:14 AM - No leftover notif
01/02/2021 1:51:55 AM - Sending notif to: cat - cid: f09c0db9-95ae-4aa2-b8a1-877fb3781b9e
01/02/2021 1:52:14 AM - Sending notif to: cat - cid: f09c0db9-95ae-4aa2-b8a1-877fb3781b9e
01/02/2021 1:53:50 AM - Sending notif to: cat - cid: f09c0db9-95ae-4aa2-b8a1-877fb3781b9e
01/02/2021 11:03:57 AM - Not found: cat - Updating to Recipients
01/02/2021 11:05:18 AM - Connected
01/02/2021 11:05:19 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:05:19 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:06:01 AM - Disconnected
01/02/2021 11:06:07 AM - Connected
01/02/2021 11:06:39 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:06:40 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:07:13 AM - Disconnected
01/02/2021 11:07:23 AM - Connected
01/02/2021 11:08:41 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:08:41 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:09:01 AM - Disconnected
01/02/2021 11:09:02 AM - Connected
01/02/2021 11:09:02 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:09:02 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:09:09 AM - Disconnected
01/02/2021 11:09:16 AM - Connected
01/02/2021 11:09:26 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:09:29 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:10:03 AM - Disconnected
01/02/2021 11:10:06 AM - Connected
01/02/2021 11:10:12 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:10:16 AM - Element '_id' does not match any field or property of class TNY.NotificationService.BUS.Models.Recipients.
01/02/2021 11:11:21 AM - Connected
01/02/2021 11:11:25 AM - Pushing leftover notif
01/02/2021 11:11:35 AM - No leftover notif
01/02/2021 11:11:52 AM - Connected
01/02/2021 11:11:57 AM - No leftover notif
01/02/2021 11:12:01 AM - No leftover notif
01/02/2021 11:12:16 AM - Disconnected
01/02/2021 11:12:21 AM - Disconnected
01/02/2021 11:12:53 AM - Not found: dog - Updating to Recipients
01/02/2021 11:13:05 AM - Connected
01/02/2021 11:13:05 AM - Pushing leftover notif
01/02/2021 11:13:05 AM - No leftover notif
01/02/2021 1:39:12 PM - Scheduled Sender running
01/02/2021 1:39:13 PM - Scheduled Sender sending
01/02/2021 1:39:15 PM - Not found: 123 - Updating to Recipients
01/02/2021 1:39:15 PM - Routine Sender sending
01/02/2021 1:39:15 PM - Routine Sender sending
01/02/2021 1:39:15 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:39:15 PM - Routine Sender sending
01/02/2021 1:40:15 PM - Scheduled Sender running
01/02/2021 1:40:15 PM - Routine Sender sending
01/02/2021 1:40:15 PM - Routine Sender sending
01/02/2021 1:40:15 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:40:15 PM - Routine Sender sending
01/02/2021 1:41:15 PM - Scheduled Sender running
01/02/2021 1:41:15 PM - Routine Sender sending
01/02/2021 1:41:15 PM - Routine Sender sending
01/02/2021 1:41:15 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:41:15 PM - Routine Sender sending
01/02/2021 1:42:15 PM - Scheduled Sender running
01/02/2021 1:42:15 PM - Routine Sender sending
01/02/2021 1:42:15 PM - Routine Sender sending
01/02/2021 1:42:15 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:42:15 PM - Routine Sender sending
01/02/2021 1:43:15 PM - Scheduled Sender running
01/02/2021 1:43:15 PM - Routine Sender sending
01/02/2021 1:43:15 PM - Routine Sender sending
01/02/2021 1:43:15 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:43:15 PM - Routine Sender sending
01/02/2021 1:46:17 PM - Scheduled Sender running
01/02/2021 1:46:17 PM - Routine Sender sending
01/02/2021 1:46:17 PM - Routine Sender sending
01/02/2021 1:46:17 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:46:17 PM - Routine Sender sending
01/02/2021 1:47:17 PM - Scheduled Sender running
01/02/2021 1:47:17 PM - Routine Sender sending
01/02/2021 1:47:17 PM - Routine Sender sending
01/02/2021 1:47:17 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:47:17 PM - Routine Sender sending
01/02/2021 1:48:17 PM - Scheduled Sender running
01/02/2021 1:48:17 PM - Routine Sender sending
01/02/2021 1:48:17 PM - Routine Sender sending
01/02/2021 1:48:17 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:48:17 PM - Routine Sender sending
01/02/2021 1:49:05 PM - Not found: dog - Updating to Recipients
01/02/2021 1:49:16 PM - Connected
01/02/2021 1:49:16 PM - Pushing leftover notif
01/02/2021 1:49:17 PM - No leftover notif
01/02/2021 1:49:18 PM - Scheduled Sender running
01/02/2021 1:49:18 PM - Routine Sender sending
01/02/2021 1:49:18 PM - Routine Sender sending
01/02/2021 1:49:18 PM - 60178cacc8684d246bf9aa49 -> Nullable object must have a value.
01/02/2021 1:49:18 PM - Routine Sender sending
01/02/2021 1:49:51 PM - Sending notif to: dog - cid: 03f9f65d-37c6-4906-9f03-ce94b341e89f
